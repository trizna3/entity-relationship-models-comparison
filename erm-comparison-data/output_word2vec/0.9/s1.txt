role vojakov-rola
typy rakiet-raketa
modely lietadiel-druh lietadla
lietadla-lietadla
vojaci-vojak
stihacie modely-stihacie lietadla
dopravne modely-dopravne
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-dopravne-druh lietadla
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-rola-vojak
