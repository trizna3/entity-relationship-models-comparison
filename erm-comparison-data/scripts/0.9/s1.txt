entity set "obranny system" {
	"lietadlo"
	"vojak"
};

entity set "lietadla" {
	"seriove cislo"
	"meno"
	"model"
	"druh"
};

entity set "vojak" {
	"meno"
	"zivot"
	"maximalny zivot"
	"rychlost"
	"sila"
	"obratnost"
	"inteligencia"
	"lietadlo"
	"rola"
};

entity set "dopravne" {
	"kapacita osob"
	"objem"
	"hmotnost materialu"
};

entity set "druh lietadla" {
	"stihacie"
	"dopravne"
};

entity set "raketa" {
	"hmotnost"
};

entity set "stihacie lietadla" {
	"pocet rakiet"
	"typy rakiet"
};

association between 
{
	"obranny system" ONE
	"lietadla" MANY
}{
};

association between 
{
	"obranny system" ONE
	"vojak" MANY
}{
};

association between 
{
	"lietadla" MANY
	"druh lietadla" ONE
}{
};

association between 
{
	"druh lietadla" MANY
	"dopravne" ONE
}{
};

association between 
{
	"druh lietadla" MANY
	"stihacie lietadla" ONE
}{
};

association between 
{
	"stihacie lietadla" MANY
	"raketa" MANY
}{
};