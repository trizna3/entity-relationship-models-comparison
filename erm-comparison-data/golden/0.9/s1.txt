modely lietadiel-druh lietadla
stihacie modely-stihacie lietadla
typy rakiet-raketa
dopravne modely-dopravne
lietadla-lietadla
vojaci-vojak
role vojakov-rola
CHANGE_CARDINALITY:S-druh lietadla-dopravne
CHANGE_CARDINALITY:S-druh lietadla-stihacie
GENERALIZATION_TO_11_ASSOCIATION:E-stihacie modely-modely lietadiel
GENERALIZATION_TO_11_ASSOCIATION:E-dopravne modely-modely lietadiel
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-rola-vojak